mne/label.py

```diff
@@ -1659,6 +1660,159 @@ def _grow_nonoverlapping_labels(subject, seeds_, extents_, hemis, vertices_,
     return labels
 
 
+def random_parcellation(subject, n_parcel, hemis, subjects_dir=None,
+                        surface='white'):
+    """Generate random cortex parcellation by growing labels.
+
+    This function generates a number of labels which don't intersect and
+    cover the whole surface. Regions are growing around randomly chosen
+    seeds.
+
+    Parameters
+    ----------
+    subject : string
+        Name of the subject as in SUBJECTS_DIR.
+    n_parcel : int
+        Total number of cortical parcels
+    hemis : array | int
```

Comment:
it should be called hemi and be a string. Following the API of pysurfer:

https://github.com/nipy/PySurfer/blob/master/surfer/viz.py#L301

Reply:
I used the doc of the function "grow_labels". If need to change it, probably should do it for grow_labels too.